SConstruct

```diff
@@ -1787,6 +1821,9 @@ if 'msi' in COMMAND_LINE_TARGETS:
 
 ### Tests ###
 if any(target.startswith('test') for target in COMMAND_LINE_TARGETS):
+    if env['python2_package'] == 'none' and env['python3_package'] == 'none':
+        env['python{}_package'.format(env['python_version'][0])] = 'minimal'
+        SConscript('interfaces/python_minimal/SConscript')
```

Comment:
I wonder whether this is insufficient to do what you presumably want, and doesn't trigger the 'build' actions in the SConscript file when you only run `scons test`.

Reply:
This is indeed insufficient, and I need to fix this still.